src/core/agent/services/multiSigService.test.ts

```diff
@@ -1150,22 +1166,24 @@ describe("Multisig sig service of agent", () => {
     jest
       .spyOn(Agent.agent.connections, "getConnectionShortDetailById")
       .mockResolvedValue(senderData);
-    jest.spyOn(Agent.agent.connections, "getConnections").mockResolvedValue([
-      {
-        id: "EHxEwa9UAcThqxuxbq56BYMq7YPWYxA63A1nau2AZ-1A",
-        connectionDate: nowISO,
-        label: "",
-        logo: "logoUrl",
-        status: ConnectionStatus.PENDING,
-      },
-      {
-        id: "EDEp4MS9lFGBkV8sKFV0ldqcyiVd1iOEVZAhZnbqk6A3",
-        connectionDate: nowISO,
-        label: "",
-        logo: "logoUrl",
-        status: ConnectionStatus.CONFIRMED,
-      },
-    ]);
+    // jest
```

Comment:
If this spy isn't needed anymore, we can remove the commented out code to clean it up :)

Reply:
sorry, I missed that